src/Models/ArduinoDeviceBase.ts

```diff
@@ -84,8 +85,11 @@ export abstract class ArduinoDeviceBase implements Device {
     return await checkExtensionAvailable(ExtensionName.Arduino);
   }
 
-  async checkPrerequisites(): Promise<boolean> {
-    return await ArduinoDeviceBase.isAvailable();
+  async checkPrerequisites(operation: string): Promise<void> {

```

Comment:
Same here as I comment in the `checkRemoteExtension` api.

Reply:
See above.